tests/datatypes/test_str.py

```diff
@@ -574,6 +574,18 @@ def test_rjust(self):
                 print(err)
             """)
 
+    def test_rpartition(self):
+        self.assertCodeExecution("""
+            Stri = "Hello World!"
+            print(Stri.rpartition("H"))
+            print(Stri.rpartition(" "))
+            print(Stri.rpartition("l"))
+            try:
+                print(Stri.rpartition(""))
+            except ValueError as err:
+                print(err)
```

Comment:
This is a good start to testing, but there's a bunch of other tests needed - most notably, partitioning on a string that doesn't exist, and partitioning on a multi-character string.